src/brsTypes/components/Poster.ts

```diff
@@ -0,0 +1,32 @@
+import { FieldModel } from "./RoSGNode";
+import { AAMember } from "./RoAssociativeArray";
+import { Group } from "./Group";
+
+export class Poster extends Group {
+    readonly defaultFields: FieldModel[] = [
+        { name: "uri", type: "string" },
+        { name: "width", type: "float", value: "0.0" },
+        { name: "height", type: "float", value: "0.0" },
+        { name: "loadSync", type: "boolean", value: "false" },
+        { name: "loadWidth", type: "float", value: "0.0" },
+        { name: "loadHeight", type: "float", value: "0.0" },
+        { name: "loadDisplayMode", type: "string", value: "noScale" },
+        { name: "loadStatus", type: "string", value: "noScale" },
+        { name: "bitmapWidth", type: "float", value: "0.0" },
+        { name: "bitmapHeight", type: "float", value: "0.0" },
+        { name: "bitmapMargins", type: "assocarray" },
+        { name: "blendColor", type: "string", value: "0xFFFFFFFF" },
+        { name: "loadingBitmapUri", type: "string" },
+        { name: "loadingBitmapOpacity", type: "float", value: "1.0" },
+        { name: "failedBitmapUri", type: "string" },
+        { name: "failedBitmapOpacity", type: "float", value: "1.0" },
+        { name: "audioGuideText", type: "string" },
+    ];
+
+    constructor(initializedFields: AAMember[] = [], readonly name: string = "Poster") {
```

Comment:
Resurrecting [this feedback](https://github.com/sjbarag/brs/pull/405#discussion_r419706979) from @sjbarag on https://github.com/sjbarag/brs/pull/405.

Do we still want to transition `AAMember[]` to `FieldModel[]`? 🤔

Reply:
Great question! So I've actually found that we _never_ use `initializedFields`, outside of the unit tests for `RoSGNode`. In theory, it makes sense:
```
<Node somefield="somevalue" />
```
Where `{ somefield: somevalue }` would be represented as an `AAMember`. Actually, though, in our current implementation, we always add the fields _after_ creating the object, (both when [the component is a child](https://github.com/sjbarag/brs/blob/master/src/brsTypes/components/RoSGNode.ts#L1197-L1216) and for `interface` fields for [custom components](https://github.com/sjbarag/brs/blob/master/src/brsTypes/components/RoSGNode.ts#L1138).

I'm not sure if this was intentional, or if we'd rather add the fields in the constructor. If the former, then we should remove `initializedFields` altogether. If that latter, then we should refactor how we create component objects and add fields.

Reply:
Either way, probably out of scope for this PR 🤷‍♂️